scripts/yaml-generator.lisp

```diff
@@ -0,0 +1,779 @@
+          (format-packages yaml-config)
+          (format-extra-deps (stack-yaml-extra-deps yaml-config))
+          (format-extra (stack-yaml-extra yaml-config))))
+
+(defun merge-group (g1 g2)
+  "merges 2 groups, taking the comment from the first"
+  (make-groups :comment (groups-comment g1)
+               :deps (append (groups-deps g1) (groups-deps g2))))
+
+;;; ----------------------------------------------------------------------
+;;; Dependencies for YAML generation
+;;; ----------------------------------------------------------------------
+
+;; --------------------------------------
+;; Crypto Style Dependencies
+;; --------------------------------------
+
+(defparameter *galois-field*
+  (make-dependency-git :name   "https://github.com/serokell/galois-field.git"
+                       :commit "576ba98ec947370835a1f308895037c7aa7f8b71"))
+
+(defparameter *galois-field-plonk*
+  (make-dependency-github :name "adjoint-io/galois-field"
+                          :commit "3b13705fe26ea1dc03e1a6d7dac4089085c5362d")
+  "Plonk uses a special version of this library")
+
+(defparameter *elliptic-curve*
+  (make-dependency-git :name   "https://github.com/serokell/elliptic-curve.git"
+                       :commit "b8a3d0cf8f7bacfed77dc3b697f5d08bd33396a8"))
+
+(defparameter *pairing*
+  (make-dependency-git :name   "https://github.com/serokell/pairing.git"
+                       :commit "cf86cf1f6b03f478a439703b050c520a9d455353"))
+
+;; --------------------------------------
+;; Tezos Style Dependencies
+;; --------------------------------------
+
+(defparameter *tezos-bake-monitor*
+  (make-dependency-git :name "https://gitlab.com/obsidian.systems/tezos-bake-monitor-lib.git"
+                       :commit "9356f64a6dfc5cf9b108ad84d1f89bcdc1f08174"
+                       :subdirs (list "tezos-bake-monitor-lib")))
+
+(defparameter *tezos-morley*
+  (make-dependency-git :name "https://gitlab.com/morley-framework/morley.git"
+                       :commit "53961f48d0d3fb61051fceaa6c9ed6becb7511e5"
+                       :subdirs (list "code/morley" "code/morley-prelude")))
+
+(defparameter *base-no-prelude-standard*
+  (string->dep-sha
+   "base-noprelude-4.13.0.0@sha256:3cccbfda38e1422ca5cc436d58858ba51ff9114d2ed87915a6569be11e4e5a90,6842")
+  "this is the standard version of base no prelude")
+
+(defparameter *base-no-prelude-special*
+  (make-dependency-git :name "https://github.com/serokell/base-noprelude.git"
+                       :commit "87df0899801dcdffd08ef7c3efd3c63e67e623c2")
+  "this is a special version of base no prelude we have to use with Michelson backend")
+
+;; --------------------------------------
+;; Stadnard Library Style Dependencies
+;; --------------------------------------
+
+(defparameter *capability*
+  (string->dep-sha "capability-0.4.0.0@sha256:d86d85a1691ef0165c77c47ea72eac75c99d21fb82947efe8b2f758991cf1837,3345"))
+
+(defparameter *extensible*
+  (make-dependency-github :name "metastatedev/extensible-data"
```

Comment:
`heliaxdev` as well?

Reply:
Probably, but I was keeping the semantics the same from the old file, thus not changing any dependency